component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/SMSOTPServiceImpl.java

```diff
@@ -276,6 +279,8 @@ private void triggerNotification(User user, String otp) throws SMSOTPException {
         }
 
         HashMap<String, Object> properties = new HashMap<>();
+        String uniqueId = getCorrelationId();
+        properties.put(Constants.UNIQUE_ID, uniqueId);
```

Comment:
```suggestion
        properties.put(Constants.UNIQUE_ID, getCorrelationId());
```
Since it has been only used once.